controllers/articles.controllers.js

```diff
@@ -0,0 +1,16 @@
+const { fetchArticleById } = require("../models/articles.models")
+
+
+function getArticleById(request, response, next) {
+    const id = request.params.article_id
+    fetchArticleById(id)
+    .then(({rows}) => {
+        const article = rows
```

Comment:
You've had to do data manipulation here - but this is probably more relevant in the model :)

Reply:
Even if it's just changing the variable name?

Reply:
Noice! Thanks